appveyor.yml

```diff
@@ -1,8 +1,5 @@
 environment:
   matrix:
-  - julia_version: 1.0
-  - julia_version: 1.1
-  - julia_version: 1.2
   - julia_version: 1.3
```

Comment:
Please keep the PR focused. Updating CI belongs in a different PR.

Reply:
I must remove julia `1.0`, `1.1`, `1.2` because 5-arguments mul! is a feature of julia 1.3.